scm/multipart.go

```diff
@@ -0,0 +1,26 @@
+package scm
```

Comment:
So this functionality is only used by stash/content. I would move this helper method in under stash 
or 
into scm/driver/stash/stash.go

Reply:
```
➜  go-scm git:(no_project) grep -irsn Multipart .
./scm/driver/bitbucket/bitbucket.go:13: "mime/multipart"
./scm/driver/bitbucket/bitbucket.go:74:                 // add the content to the multipart
./scm/driver/bitbucket/bitbucket.go:77:                 w := multipart.NewWriter(&b)
./scm/driver/bitbucket/bitbucket.go:95:                 // write the multipart response to the body
./scm/driver/bitbucket/bitbucket.go:97:                 // write the content type that contains the length of the multipart
./scm/driver/bitbucket/bitbucket.go:103:                        writer := multipart.NewWriter(body)
```
nothing else is using it yet. I would definitely make it only for stash for now. Until another git provider uses it